OpenRA.Mods.Common/Projectiles/Bullet.cs

```diff
@@ -96,10 +105,12 @@ public class Bullet : IProjectile, ISync
 		ContrailRenderable contrail;
 		string trailPalette;
 
-		[Sync] WPos pos, target;
+		[Sync] WPos pos, target, source;
 		int length;
+		int distance;
 		[Sync] int facing;
 		int ticks, smokeTicks;
+		int bounceCount;
```

Comment:
This name implies it should be counting up, but it is actually the number of bounces remaining.  Can you please rename this to `remainingBounces` or something similar?